azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -98,7 +98,7 @@ If your nodes were configured via Virtual Machine Manager (VMM), those configura
 
 ### Step 5: Start the Network ATC service
 
-As a precaution, to control the speed of the rollout, we paused the node and then stopped or disabled the Network ATC service in the previous steps. Since Network ATC intents are implemented cluster-wide, perform this step only once.
+As a precaution, to control the speed of the rollout, we paused the node and then stopped and disabled the Network ATC service in the previous steps. Since Network ATC intents are implemented cluster-wide, perform this step only once.
```

Comment:
@helmste - We stopped or disabled depending on how the reviewer would refer to the action. So, I think it's okay to leave this as it is.

Reply:
@ronmiab - the commands in the 'Stop the Network ATC service' do both actions (disable and stop).  I think this should reflect both actions since both need to be performed.  Stopping the service prevents the changes from propagating but disabling is an additional step that prevents the service from automatically restarting and propagating changes should the node be rebooted before actions are completed.